main.py

```diff
@@ -42,25 +58,29 @@ def prompt_redo():
 
 
 print("At any time, type EXIT to stop. ")
-while True:
-    mode = str(input("What mode do you wish to use? FLIP == flip text, CASE == change casing. "))
+while i < 1:
+    mode = str(input("What mode do you wish to use? FLIP == flip text, CASE == change casing ENCHANT == Convert text to the standard galactic alphabet used by the Minecraft enchanting table "))
     text_input = str(input("Input the string you need to convert here: "))
     if mode.lower() == "case":
         casing = int(input("Type 1 for uppercase, 2 for lowercase. "))
-        if casing == 1:
-            case_switch(casing, text_input)
-            restart = prompt_redo()
-            if restart == "stop":
-                break
-        elif casing == 2:
-            case_switch(casing, text_input)
-            restart = prompt_redo()
-            if restart == "stop":
-                break
+        case_switch(casing, text_input)
+        restart = prompt_redo()
+        if restart == "stop":
+            break
+
     elif mode.lower() == "flip":
         text_flip(text_input)
         restart = prompt_redo()
         if restart == "stop":
             break
+
+    elif mode.lower() == "enchant":
+        enchant_text(text_input)
+        restart = prompt_redo()
+        if restart == "stop":
+            break
+
+    elif mode.lower() == "EXIT":
+        i = 1
```

Comment:
```suggestion
        break
```
Since we don't use i < 1 anymore, use `break` instead.